Rocket.toml.dist

```diff
@@ -6,6 +6,9 @@ instance_name = "Freifunk ... Knotenüberwachung"
 instance_article_dative = "der"
 # The sender address of the emails that are sent by ff-node-monitor.
 email_from = "...@...freifunk.net"
+# When fewer nodes than this are online in the nodes.json file, the database won't be updated
+# and there will be no warning mails sent.
```

Comment:
```suggestion
# and there will be no warning mails sent. A value of 0 disables this sanity check.
```

Reply:
I would say, it is not a "sanity check", it is a prevention to inform all users that their node is offline, although the problem is not on their side and with this option and also you will not loose the data any more which nodes were already offline before this fail

Reply:
Yeah, that sounds like a sanity check to me.